timber/src/main/java/timber/log/Timber.java

```diff
@@ -594,7 +595,11 @@ protected String createStackElementTag(StackTraceElement element) {
         tag = m.replaceAll("");
       }
       tag = tag.substring(tag.lastIndexOf('.') + 1);
-      return tag.length() > MAX_TAG_LENGTH ? tag.substring(0, MAX_TAG_LENGTH) : tag;
+      // Tag length limit was removed in API 24.
+      if (tag.length() <= MAX_TAG_LENGTH || Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
```

Comment:
static import?

Reply:
I would have had it required a line break, but otherwise /shrug